kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -554,6 +554,35 @@ public void testGetCaseInstancesWithData() {
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
 
+    @Test
+    public void testByPassAuthCaseTasksAssignedAsPotentialOwner() throws Exception {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+        try {
+            List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
+            assertEquals("Hello1", tasks.get(0).getName());
+
+            changeUser(USER_JOHN);
+
+            tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_MARY, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(0, tasks.size());
+
+            tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
```

Comment:
as suggested above, after calling changeUser, I would now add and check again the tasks assigned when an empty user is sent in the request (so the tasks assigned to John are retrieved - 0 in this case again):

```suggestion
            tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, "", 0, 10);
            assertEquals(0, tasks.size());
```